ui-participant/src/hub/survey/SurveyView.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import { generateThreePageSurvey } from 'test-utils/test-survey-factory'
+import { PageNumberControl, useSurveyJSModel } from 'util/surveyJsUtils'
+import { render, screen } from '@testing-library/react'
+import { SurveyFooter } from './SurveyView'
+import { usePortalEnv } from 'providers/PortalProvider'
+import { Survey } from '@juniper/ui-core'
+
+jest.mock('providers/PortalProvider', () => ({ usePortalEnv: jest.fn() }))
+beforeEach(() => {
+  // @ts-expect-error TS doesn't realize this function is mocked
+  usePortalEnv.mockReturnValue({
```

Comment:
I see other places in the code using `as jest.Mock` to handle this.
```suggestion
  (usePortalEnv as jest.Mock).mockReturnValue({
```